src/components/existing-request/nr-approved-gray-box.vue

```diff
@@ -55,11 +55,11 @@
         class="d-flex justify-center my-1"
       >
         <v-btn
-          v-if="showOpenExternalIcon"
+          v-if="showOpenExternalIcon && !isSupportedAmalgamation"
```

Comment:
I'm not understanding why you need `&& !isSupportedAmalgamation`, since that appears to alreay be part of the logic for showOpenExternalIcon (line 221).

In fact, I think `isSupportedAmalgamation` isn't doing what you think it is -- in the code below, it's a function, but here you don't pass any parameters (or use brackets), so this code is actually testing for the presence of the function (not its result).